### ISSUE TITLE
Proposed Analysis: quantify telomerase activity across pediatric brain tumors

### ISSUE THREAD

**Issue by syzheng** [AlexsLemonade/OpenPBTA-analysis#148]
#### Scientific goals

_The goal is to quantify telomerase activity and correlate them with telomere length and molecular alterations (TERTp mutation, ATRX mutation, etc)_

#### Proposed methods

_We will use our newly developed method EXTEND (EXpression based Telomerase ENzymatic activity Detection)_

#### Required input data

_Gene expression from RNAseq (either of TPM, RPKM, or counts)_

#### Proposed timeline

_One to two weeks._

#### Relevant literature

_Barthel et al. Nat Genet, 2017; Zheng et al. Cancer Cell, 2016; Ackermann et al. Science, 2016_

**Comment by cgreene** [comment]
This sounds exciting! As I think about potential caveats, does it matter if the RNA-seq samples are poly-A selected or rRNA depleted?

**Comment by syzheng** [comment]
> This sounds exciting! As I think about potential caveats, does it matter if the RNA-seq samples are poly-A selected or rRNA depleted?

that is a great point. we currently use data from regular polyA enriched protocol, mostly because our primary input data is TCGA. this does impact the method, because a key gene in our signature, TERC, is a non-coding RNA that is not properly captured by polyA methods. PCR shows this gene is abundantly expressed across tissues; however, RNAseq data from TCGA and GTEx only show very low expression of this gene. EXTEND demonstrates reasonable performance with both TCGA, CCLE and GTEx, but We have not tested data from total RNAseq or rRNA depletion. Great point.

**Comment by cgreene** [comment]
@syzheng : Ok! This dataset contains both poly-A and rRNA depleted samples. #120 and https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/selection-strategy-comparison took a dive into the implications for gene expression analyses based on some earlier work by @cbethell.

I'm a bit confused by "a key gene in our signature, TERC, is a non-coding RNA that is not properly captured by polyA methods" and also "we currently use data from regular polyA enriched protocol, mostly because our primary input data is TCGA". Did you mean that you are better off with poly-A? There are many fewer poly-A samples here than rRNA-depleted, and @jharenza is looking to determine whether or not we can generate some that are matched (sequenced with both protocols).

**Comment by syzheng** [comment]
> @syzheng : Ok! This dataset contains both poly-A and rRNA depleted samples. #120 and https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/selection-strategy-comparison took a dive into the implications for gene expression analyses based on some earlier work by @cbethell.
> 
> I'm a bit confused by "a key gene in our signature, TERC, is a non-coding RNA that is not properly captured by polyA methods" and also "we currently use data from regular polyA enriched protocol, mostly because our primary input data is TCGA". Did you mean that you are better off with poly-A? There are many fewer poly-A samples here than rRNA-depleted.
> 
> As something that may be helpful in extending an analysis across both sets: @jharenza is looking to determine whether or not we can generate some that are matched (sequenced with both protocols).

EXTEND was developed using data from polyA. We essentially do not know if it works for rRNA depletion, because we did not have this type of data when we benchmarked the method. The key is TERC, a non-coding RNA that constitutes our gene signature as well as the telomerase complex. It would be great if we have a few cases that are sequenced by both methods. Otherwise, we can examine the distribution of TERC expression in the dataset to see if they behave similarly to poly A datasets.

**Comment by cgreene** [comment]
Gotcha! You'll find both sets of files in the data download as processed in a few different ways:
```
pbta-gene-expression-kallisto.polya.rds
pbta-gene-expression-kallisto.stranded.rds
pbta-gene-expression-rsem-fpkm.polya.rds
pbta-gene-expression-rsem-fpkm.stranded.rds
pbta-gene-counts-rsem-expected_count.polya.rds
pbta-gene-counts-rsem-expected_count.stranded.rds
```
For now, it will be interesting to see if the distribution is different and/or if TERC matches the estimates from the method in the `stranded` ones. Hopefully we'll have the set with both in the not terribly distant future, but we shouldn't wait for them to get started. Thanks for taking this on!

**Comment by jharenza** [comment]
Hi @syzheng ! Checking in on this analysis - do you have an idea of when you or your team would file a PR for this? Thanks!

**Comment by syzheng** [comment]
yes, we have finished the score calculation. will update on github once we
have more on integration.
Siyuan

**Comment by jharenza** [comment]
Hi @syzheng! Wanted to update you that with V12 of the data release, we will provide stranded seq for 45 samples on which we also have polyA rna-seq, so would be interesting to determine whether there are telomerase prediction differences in these two sets of data. Stay tuned end of this week/early next week. Also looking forward to your PR!

**Comment by syzheng** [comment]
Thanks for the heads up! I will update the group by next week.
best,

**Comment by jharenza** [comment]
Hi @syzheng ! Happy New Year! Do you think your team will be able to submit a PR on this analysis sometime soon? We are starting to wrap up/finalize analyses and determine manuscript figures. Thanks!

**Comment by syzheng** [comment]
sorry. yes, I will make sure to complete it next few days.

**Comment by jharenza** [comment]
No worries, glad to hear!

**Comment by jaclyn-taroni** [comment]
Addressed through #494, #506, #511, and #516
